charts/zookeeper/values.yaml

```diff
@@ -10,6 +15,11 @@ triggerRollingRestart: false
 
 domainName:
 labels: {}
+
```

Comment:
Below code block is not required looks like, service account is mentioned at https://github.com/pravega/zookeeper-operator/blob/fe02e094a4c2f640dbecd56d4424cc1b58c86d25/charts/zookeeper/values.yaml#L19

Reply:
It is. It's used here:
https://github.com/klarrio-kpn-dsh/zookeeper-operator/blob/fe02e094a4c2f640dbecd56d4424cc1b58c86d25/charts/zookeeper/templates/post-install-upgrade-hooks.yaml#L41

And in line with the `zookeeper-operator` values:
https://github.com/klarrio-kpn-dsh/zookeeper-operator/blob/fe02e094a4c2f640dbecd56d4424cc1b58c86d25/charts/zookeeper-operator/values.yaml#L27

Reply:
referring zookeeper/values.yaml not the zookeeper-operator

Reply:
Line 56 here has `pod.imagePullSecrets` (used on ZookeeperCluster resource).
The proposed addition would specify `serviceAccount.imagePullSecrets` (to be used on the hooks).
@TomBillietKlarrio Why don't we move this to `hooks.serviceAccount.imagePullSecrets` to make it clear that the particular set of pull secrets would be pertinent to the hook image docker registry?